esp/services/ws_workunits/ws_workunitsHelpers.hpp

```diff
@@ -145,7 +145,9 @@ struct WsWUExceptions
 #define WUINFO_All                      0xFFFFFFFF
 
 static constexpr unsigned defaultMaxLogRecords = 10000;
-static constexpr unsigned defaultWULogSearchTimeBufferSecs = 600;
+static constexpr unsigned defaultWULogSearchTimeBufferSecs = 43200; // +/- 12 hours from WU creation
+                                                                    // Accounts for potential TZ discrepancy
+                                                                    // TZs and logic based on TZs should normalize to UTC instead
```

Comment:
@jakesmith did you miss these comments, or did you envision different wording?

Reply:
hm, there was only 1 commit when I looked - the review status had been refreshed, but github only showed 1 commit.
Odd.

Reply:
all good, thanks @jakesmith